chromadb/utils/embedding_functions/ollama_embedding_function.py

```diff
@@ -34,7 +29,7 @@ def __init__(self, url: str, model_name: str) -> None:
         self._model_name = model_name
         self._session = requests.Session()
 
-    def __call__(self, input: Documents) -> Embeddings:
+    def __call__(self, input: Union[Documents, str]) -> Embeddings:
```

Comment:
`input if isinstance(input, list) else [input]` is allowing for things that are not lists subclasses.